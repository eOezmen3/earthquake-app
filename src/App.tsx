import Earthquakes from './components/Earthquakes';
import Footer from './components/Footer';
import React from 'react';

function App() {
  return (
    <>
      <Earthquakes />
      <Footer />
    </>
  );
}

export default App;
