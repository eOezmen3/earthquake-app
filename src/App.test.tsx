import '@testing-library/jest-dom';

import App from './App';
import React from 'react';
import { render } from '@testing-library/react';

describe('Earthquakes', () => {
  test('renders App', () => {
    const { asFragment } = render(<App />);
    expect(asFragment()).toMatchSnapshot();
  });
});
