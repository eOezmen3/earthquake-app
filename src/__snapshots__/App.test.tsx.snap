// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Earthquakes renders App 1`] = `
<DocumentFragment>
  <div
    class="map leaflet-container leaflet-touch leaflet-grab leaflet-touch-drag leaflet-touch-zoom"
    style="position: relative;"
    tabindex="0"
  >
    <div
      class="leaflet-pane leaflet-map-pane"
      style="left: 0px; top: 0px;"
    >
      <div
        class="leaflet-pane leaflet-tile-pane"
      >
        <div
          class="leaflet-layer "
          style="z-index: 1;"
        >
          <div
            class="leaflet-tile-container leaflet-zoom-animated"
            style="z-index: 18; left: 0px; top: 0px;"
          >
            <img
              alt=""
              class="leaflet-tile"
              role="presentation"
              src="https://c.basemaps.cartocdn.com/dark_all/2/1/1.png"
              style="width: 256px; height: 256px; left: -256px; top: -256px;"
            />
            <img
              alt=""
              class="leaflet-tile"
              role="presentation"
              src="https://a.basemaps.cartocdn.com/dark_all/2/2/1.png"
              style="width: 256px; height: 256px; left: 0px; top: -256px;"
            />
            <img
              alt=""
              class="leaflet-tile"
              role="presentation"
              src="https://a.basemaps.cartocdn.com/dark_all/2/1/2.png"
              style="width: 256px; height: 256px; left: -256px; top: 0px;"
            />
            <img
              alt=""
              class="leaflet-tile"
              role="presentation"
              src="https://b.basemaps.cartocdn.com/dark_all/2/2/2.png"
              style="width: 256px; height: 256px; left: 0px; top: 0px;"
            />
          </div>
        </div>
      </div>
      <div
        class="leaflet-pane leaflet-overlay-pane"
      />
      <div
        class="leaflet-pane leaflet-shadow-pane"
      />
      <div
        class="leaflet-pane leaflet-marker-pane"
      />
      <div
        class="leaflet-pane leaflet-tooltip-pane"
      />
      <div
        class="leaflet-pane leaflet-popup-pane"
      />
    </div>
    <div
      class="leaflet-control-container"
    >
      <div
        class="leaflet-top leaflet-left"
      >
        <div
          class="leaflet-control-zoom leaflet-bar leaflet-control"
        >
          <a
            aria-disabled="false"
            aria-label="Zoom in"
            class="leaflet-control-zoom-in"
            href="#"
            role="button"
            title="Zoom in"
          >
            <span
              aria-hidden="true"
            >
              +
            </span>
          </a>
          <a
            aria-disabled="false"
            aria-label="Zoom out"
            class="leaflet-control-zoom-out"
            href="#"
            role="button"
            title="Zoom out"
          >
            <span
              aria-hidden="true"
            >
              âˆ’
            </span>
          </a>
        </div>
      </div>
      <div
        class="leaflet-top leaflet-right"
      />
      <div
        class="leaflet-bottom leaflet-left"
      />
      <div
        class="leaflet-bottom leaflet-right"
      >
        <div
          class="leaflet-control-attribution leaflet-control"
        >
          <a
            href="https://leafletjs.com"
            title="A JavaScript library for interactive maps"
          >
            Leaflet
          </a>
        </div>
      </div>
    </div>
  </div>
  <div
    class="footer"
  >
    <h5>
      Made by Emre Oezmen for Futurice Interview
    </h5>
  </div>
</DocumentFragment>
`;
